Lombiq.Hosting.Azure.ApplicationInsights/Extensions/ApplicationInsightsInitializerExtensions.cs

```diff
@@ -43,6 +36,20 @@ public static class ApplicationInsightsInitializerExtensions
             .GetSection("OrchardCore:Lombiq_Hosting_Azure_ApplicationInsights");
         applicationInsightsConfigSection.Bind(applicationInsightsOptions);
 
+        services.Configure<TelemetryConfiguration>(config =>
+        {
+            if (applicationInsightsOptions.EnableLocalDevelopment)
+            {
+                var credential = new DefaultAzureCredential();
```

Comment:
Won't this try all, including `ManagedIdentityCredential`? https://learn.microsoft.com/en-us/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet

How did you test this module before this change locally?

Reply:
Changed this to use `DefaultAzureCredential` this was described a bit weird on the ms docs.